### PR TITLE
Export Northstar version number in DLL

### LINKED ISSUE #455: Export Northstar version number in DLL as well
Version number is only exported in Launcher but it should be exported in DLL as well.

image

Also helpful with #451

### PR DESCRIPTION
**Pull request by ASpoonPlaysGames**
closes #455 

will conflict with #541 as the workflow that #541 edits will need the same change added to it

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Given that @pg9182 requested to be pinged on merge of #541 I'd say we merge this PR first ^^

File: NorthstarDLL/resource1.h
Comment (GeckoEidechse) [inline NorthstarDLL/resource1.h:1]: I assume this file is more or less a copy/paste of the NorthstarLauncher version?

File: NorthstarDLL/resources.rc
Comment (GeckoEidechse) [inline NorthstarDLL/resources.rc:1]: I assume this file is more or less a copy/paste of the NorthstarLauncher version?

File: NorthstarDLL/resource1.h
Comment (ASpoonPlaysGames) [reply NorthstarDLL/resource1.h:1]: Pretty much, yeah

File: NorthstarDLL/resources.rc
Comment (ASpoonPlaysGames) [reply NorthstarDLL/resources.rc:1]: Pretty much, yeah

**Review by GeckoEidechse — APPROVED** [review]
**Testing:**
Confirmed working in testing

image

**Code review:**
As for code review, I'm trusting @ASpoonPlaysGames when he said that he basically just copied the same file from the launcher resources and adjusted it.

**Pull request merged**